tenable/sc/organization.py

```diff
@@ -0,0 +1,179 @@
+                'name': i.get('name')
+            } for i in self._check(
+                'ip_info_links', kw['ip_info_links'], list, default=[]
+            ) if i.get('link')]
+            del kw['ip_info_links']
+
+        if 'vuln_score_low' in kw:
+            kw['vulnScoreLow'] = self._check(
+                'vuln_score_low', kw['vuln_score_low'], int, default=1)
+            del kw['vuln_score_low']
+
+        if 'vuln_score_medium' in kw:
+            kw['vulnScoreMedium'] = self._check(
+                'vuln_score_medium', kw['vuln_score_medium'], int, default=3)
+            del kw['vuln_score_medium']
+
+        if 'vuln_score_high' in kw:
+            kw['vulnScoreHigh'] = self._check(
+                'vuln_score_high', kw['vuln_score_high'], int, default=10)
+            del kw['vuln_score_high']
+
+        if 'vuln_score_critical' in kw:
+            kw['vulnScoreCritical'] = self._check(
+                'vuln_score_critical', kw['vuln_score_critical'], int, default=40)
+            del kw['vuln_score_critical']
+
+        return kw
+
+    def create(self, **kw):
+        kw = self._constructor(**kw)
+        return self._api.post('organization', json=kw).json()['response']
+
+    def list(self, fields=None):
+        '''
+        Retrieves a list of organizations.
+
+        + `SC organization List <https://docs.tenable.com/sccv/api/Organization.html#OrganizationRESTReference-/organization>`_
+
+        Args:
+            fields (list, optional):
+                The list of fields that are desired to be returned.  For details
+                on what fields are available, please refer to the details on the
+                request within the organization list API doc.
+            repo_type (str, optional):
+                Restrict the response to a specific type of organization.  If not
+                set, then all organization types will be returned.  Allowed types
+                are ``All``, ``Local``, ``Remote``, and ``Offline``.
+
+        Returns:
+            list: List of organization definitions.
+
+        Examples:
+            Retrieve all of all of the organizations:
+
+            >>> repos = sc.organizations.list()
+
+            Retrieve all of the remote organizations:
+
+            >>> repos = sc.organizations.list(repo_type='Remote')
+        '''
+        params = dict()
+        if fields:
+            params['fields'] = ','.join([self._check('field', f, str)
+                                         for f in fields])
+        return self._api.get('organization', params=params).json()['response']
+
+    def details(self, id, fields=None):
+        params = dict()
+        if fields:
+            params['fields'] = ','.join([self._check('field', f, str) for f in fields])
+
+        return self._api.get('organization/{}'.format(
+            self._check('id', id, int)), params=params).json()['response']
+
+    def edit(self, org_id, **kw):
+        '''Updates an existing organization
```

Comment:
Please complete the docstring.  Add the Args and Returns statements

Reply:
done